roles/openshift_metrics/templates/metrics-server-deployment.j2

```diff
@@ -0,0 +1,49 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: metrics-server
+  namespace: "{{ openshift_metrics_server_project }}"
+  labels:
+    k8s-app: metrics-server
+    kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: Reconcile
+    version: "{{openshift_metrics_server_version}}"
+    metrics-infra: metrics-server
+spec:
+  selector:
+    matchLabels:
+      k8s-app: metrics-server
+      version: "{{openshift_metrics_server_version}}"
```

Comment:
hmm... do we really want version in the selector here?
(ideally, update should be a rolling deployment update)

Reply:
> do we really want version in the selector here?

Nope; this can go.

Reply:
> do we really want version in the selector here?

I now recall why this is here; I copied this [setup](https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/metrics-server/metrics-server-deployment.yaml#L32)